### PR TITLE
Expose timestamp on installations and make members async 

### PR DESCRIPTION
**Pull request by codabrink**
To make revoking installation keys more useful lets also expose when the installation was created at. This however requires members to be an async function now.

### CODE REVIEW

File: example/src/main/java/org/xmtp/android/example/conversation/ConversationViewHolder.kt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package org.xmtp.android.example.conversation
 
 import androidx.recyclerview.widget.RecyclerView
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.launch
```
Comment (nplasterer) [inline example/src/main/java/org/xmtp/android/example/conversation/ConversationViewHolder.kt:5]: These should get removed

File: library/src/androidTest/java/org/xmtp/android/library/GroupTest.kt [hunk 2]
```diff
@@ -84,8 +84,8 @@ class GroupTest {
             alixGroup.addMembers(listOf(caro.walletAddress))
             boGroup.sync()
         }
-        assertEquals(alixGroup.members().size, 3)
-        assertEquals(boGroup.members().size, 3)
+        assertEquals(alixGroup.members(alixClient).size, 3)
+        assertEquals(boGroup.members(boClient).size, 3)
```
Comment (nplasterer) [inline library/src/androidTest/java/org/xmtp/android/library/GroupTest.kt:88]: members shouldn't take a client.

File: library/src/main/java/org/xmtp/android/library/Conversation.kt [hunk 3]
```diff
@@ -56,18 +57,17 @@ sealed class Conversation {
...
         }
 
-    val peerAddresses: List<String>
-        get() {
-            return when (this) {
-                is V1 -> listOf(conversationV1.peerAddress)
-                is V2 -> listOf(conversationV2.peerAddress)
-                is Group -> group.peerInboxIds()
-            }
+    suspend fun peerAddresses(): List<String> {
```
Comment (nplasterer) [inline library/src/main/java/org/xmtp/android/library/Conversation.kt:64]: This shouldn't be suspend.

File: library/src/main/java/org/xmtp/android/library/Conversation.kt [hunk 4]
```diff
@@ -102,7 +102,7 @@ sealed class Conversation {
      * @return [TopicData] that contains all the information about the Topic, the conversation
      * context and the necessary encryption data for it.
      */
-    fun toTopicData(): TopicData {
+    suspend fun toTopicData(): TopicData {
```
Comment (nplasterer) [inline library/src/main/java/org/xmtp/android/library/Conversation.kt:105]: This shouldnt be suspend.

File: library/src/androidTest/java/org/xmtp/android/library/GroupTest.kt (hunk 2 above)
Comment (nplasterer) [inline library/src/androidTest/java/org/xmtp/android/library/GroupTest.kt:88]: If your trying to get around the fact that members are now suspend just wrap all this stuff in a `runBlocking`

```suggestion
        runBlocking {
          assertEquals(alixGroup.members().size, 3)
          assertEquals(boGroup.members().size, 3)
        }
```

**Review by nplasterer — COMMENTED** [review]
The tests need some work. Can show you this week how to run them.

**Review by neekolas — APPROVED** [review]

**Pull request merged**